src/components/views/game/game/scoreannouncing/AnnouncementModal.tsx

```diff
@@ -152,11 +152,15 @@ export const AnnouncementModal = observer(() => {
                 </div>
                 <Tooltip
                   title={
-                    "Here you can choose how many tricks you think you will win!"
-                  }
+                    !active
```

Comment:
i would have but the tooltip on the buttons themselves, and then you know which error message you must display

Reply:
I tried but it as of right now, it doe's not show the tooltip if assigned to the buttons. 
I think this change would be suited for the Issue: https://github.com/sopra-fs22-group-36/screw-your-neighbor-react/issues/120 QOL changes